pilot/pkg/bootstrap/server.go

```diff
@@ -343,9 +343,15 @@ func NewServer(args *PilotArgs, initFuncs ...func(*Server)) (*Server, error) {
 	// Notice that the order of authenticators matters, since at runtime
 	// authenticators are activated sequentially and the first successful attempt
 	// is used as the authentication result.
-	authenticators := []security.Authenticator{
-		&authenticate.ClientCertAuthenticator{},
+	authenticators := []security.Authenticator{}
```

Comment:
I remember doing this in my PR - not sure why it is not there. May be I reverted in later commits